test/test.js

```diff
@@ -3,64 +3,76 @@ var assert = require('assert');
 var getWindow = require('../');
 var getDocument = require('get-document');
 
+var inNode = require('detect-node');
+
+// Zuul is configured to pass on to Browserify a configuration to exclude the
+// following module, 'jsdom' because it is only for testing in node. Browsers
+// have real dom
+var jsdom = inNode ? require('jsdom') : null;
+
+var doc = inNode ? jsdom.jsdom() : document;
+var win = inNode ? doc.parentWindow : window;
```

Comment:
Also, `win` was used already, throughout the tests, as the return value from `getWindow` so I took the liberty to change it to `result` throughout.